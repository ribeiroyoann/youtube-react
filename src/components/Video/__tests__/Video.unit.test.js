import React from "react";
import { Video } from "../Video";
import { shallow } from "enzyme";

test("renders video component correctly", () => {
  const wrapper = shallow(<Video id="BtyHYIpykN0" />);
  expect(wrapper).toMatchSnapshot();
});

test("renders null if no video id", () => {
  const wrapper = shallow(<Video />);
  expect(wrapper).toMatchSnapshot();
});
