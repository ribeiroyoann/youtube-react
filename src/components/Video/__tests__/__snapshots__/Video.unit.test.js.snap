// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders null if no video id 1`] = `""`;

exports[`renders video component correctly 1`] = `
<div
  className="video-container"
>
  <div
    className="video"
  >
    <iframe
      allow="autoplay; encrypted-media"
      allowFullScreen={true}
      frameBorder="0"
      height="100%"
      src="https://www.youtube.com/embed/BtyHYIpykN0"
      title="video"
      width="100%"
    />
  </div>
</div>
`;
